core/client/fs/src/main/java/alluxio/client/file/cache/PageStore.java

```diff
@@ -93,12 +140,14 @@ default int get(PageId pageId, byte[] buffer) throws IOException, PageNotFoundEx
    * @param bytesToRead bytes to read in this page
    * @param buffer destination buffer
    * @param bufferOffset offset in buffer
+   * @param isTemporary is page data temporary
    * @return the number of bytes read
    * @throws IOException when the store fails to read this page
    * @throws PageNotFoundException when the page isn't found in the store
    * @throws IllegalArgumentException when the page offset exceeds the page size
    */
-  int get(PageId pageId, int pageOffset, int bytesToRead, byte[] buffer, int bufferOffset)
+  int get(PageId pageId, int pageOffset, int bytesToRead, byte[] buffer, int bufferOffset,
```

Comment:
Generally speaking i think it's cleaner to have separate API for getPage and getTemporaryPage. Just food for thoughts.